python/setup.py

```diff
@@ -28,6 +28,7 @@
           file=sys.stderr)
     sys.exit(-1)
 
+__version__ = "Unknown"  # Prevent linters from raising 'undefined name'
```

Comment:
Shall we simply noqa for `VERSION = __version__`? looks weird we have a version defined as Unknown to fix lint errors.